<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkingWindow</class>
 <widget class="QMainWindow" name="WorkingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>525</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>PyEasier</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 173);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="begin_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>130</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正清刻本悦宋简体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始训练!</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="choiceQ_cbx">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>100</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正小标宋简体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择题</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="judgementQ_cbx">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>100</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正小标宋简体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>判断题</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="shortAnsQ_cbx">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>100</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正小标宋简体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>简答题</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="welcome_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正小标宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>又来做题啦，彦祖！</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="questionNum_sbx">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="welcome_pic">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>0</y>
      <width>425</width>
      <height>502</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="fillinBlankQ_cbx">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>100</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正小标宋简体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>填空题</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="wrongSet_cbx">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正小标宋简体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>易错模式</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>个人中心</string>
    </property>
    <addaction name="actionUserInf"/>
    <addaction name="actionAnsSta"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>退出登录</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuUser"/>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>退出登录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUserInf">
   <property name="text">
    <string>我的主页</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAnsSta">
   <property name="text">
    <string>做题情况</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
